components/toolbar/index.js

```diff
@@ -30,6 +30,7 @@ function Toolbar( { controls } ) {
 						'is-active': control.isActive,
 					} ) }
 					aria-pressed={ control.isActive }
+					focus={ focus && ! index }
```

Comment:
This line is causing some trouble for me in tests.  I am unsure as to what is happening here.  Where is `focus` coming from?  Is this supposed to be passed into the component as a `prop`?  If that is the case, we will need to add focus to the function signature at the top.  `Toolbar( { controls, focus } )`.